test/e2e/index.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { chromium, firefox } from 'playwright'
+import { BASE_URL, CONTENT_SELECTOR, PAGE_TITLE, TITLE_SELECTOR } from './constants'
```

Comment:
Not sure that the selector and title values should be constants as they appear like they should be specific to this test?